src/main/java/seedu/address/MainApp.java

```diff
@@ -103,45 +98,37 @@ public void init() throws Exception {
      * or an empty book will be used instead if errors occur when reading {@code storage}'s address book.
      */
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
-        Optional<ReadOnlyAddressBook> addressBookOptional;
         Optional<ReadOnlyBidBook> bidBookOptional;
         Optional<ReadOnlyBidderAddressBook> bidderAddressBookOptional;
         Optional<ReadOnlySellerAddressBook> sellerAddressBookOptional;
         Optional<ReadOnlyPropertyBook> propertyBookOptional;
         Optional<ReadOnlyMeetingBook> meetingBookOptional;
 
-        ReadOnlyAddressBook initialData;
         ReadOnlyBidBook initialBidData;
         ReadOnlyBidderAddressBook initialBidderData;
         ReadOnlySellerAddressBook initialSellerData;
         ReadOnlyPropertyBook initialPropertyData;
         ReadOnlyMeetingBook initialMeetingData;
 
         try {
-            addressBookOptional = storage.readAddressBook();
             bidBookOptional = storage.readBidBook();
             propertyBookOptional = storage.readPropertyBook();
             bidderAddressBookOptional = storage.readBidderAddressBook();
             sellerAddressBookOptional = storage.readSellerAddressBook();
             meetingBookOptional = storage.readMeetingBook();
 
-            if (!addressBookOptional.isPresent()) {
-                logger.info("AddressBook Data file not found. Will be starting with a sample AddressBook");
-            }
             if (!bidBookOptional.isPresent()) {
                 logger.info("BidBook Data file not found. Will be starting with a sample BidBook");
             }
             if (!bidderAddressBookOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample bidderAddressBook");
+                logger.info("Bidder Data file not found. Will be starting with a sample bidderAddressBook");
             }
             if (!sellerAddressBookOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample sellerAddressBook");
+                logger.info("Seller Data file not found. Will be starting with a sample sellerAddressBook");
             }
             if (!meetingBookOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample meetingBook");
+                logger.info("Meeting Data file not found. Will be starting with a sample meetingBook");
```

Comment:
I think it shud be BidderAddressBook file not found. or MeetingBook cause it the bidderbook thats not found not the data on the bidders